extension/src/popup/components/manageAssets/ChooseAsset/index.tsx

```diff
@@ -94,7 +94,7 @@ export const ChooseAsset = ({ balances }: ChooseAssetProps) => {
           });
 
           // disable soroban payment until we can get the new SDK working
-          const disableSorobanPayment = true;
+          const disableSorobanPayment = false;
```

Comment:
can you please just remove this feature flag altogether now? I just added it to quickly block token payments

Reply:
oh right, removing